library_generation/generate_composed_library.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# This script allows generation of libraries that are composed of more than one
+# service version. It is achieved by calling `generate_library.sh` without
+# postprocessing for all service versions and then calling
+# postprocess_library.sh at the end, once all libraries are ready.
+#
+# Arguments
+# --generation_queries: a single string of key-value groups separated by a
+# pipe |. They key-value groups are in the form of `key=value` and will
+# be converted to an argument to generate_library.sh (`--key value`).
+#   example: "key1=value1 key2=value2,,,key1=value3 key2=value4"
+#   In this case, generate_library.sh will be called once with value1 and value2
+#   and once with value3 and value4.
+# --versions_file: list of versions to be applied to the pom.xml and readmes
+#
+# Note: googleapis repo is found in https://github.com/googleapis/googleapis
+#
+# Prerequisites
+# - Needs an `output` folder at the location of the calling shell
```

Comment:
Does `generate_library.sh` create a `output` folder by default?

Reply:
yes, but it will need some of the folders explained below. This is just to explain that `output` should contain a few extra dependency folders (e.g. protos in `google` folder). 

It will however assume the output folder is located in `$(pwd)/output` or so (see line `32`).

I expanded this comment to clarify this